tensorbay/opendataset/HKD/loader.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT Licence.
+#
+# pylint: disable=invalid-name
+
+"""Dataloader for Finger Tapping Hand Keypoints Dataset(HKD)."""
+import csv
+import os
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.geometry import Keypoint2D
+from tensorbay.label import LabeledKeypoints2D
+
+DATASET_NAME = "HKD"
+csv_names = {
```

Comment:
The name of constants should be in the upper case. And constants used only here should be private.